pkg/controller/deployment/deployment_controller.go

```diff
@@ -744,3 +886,13 @@ func (dc *DeploymentController) updateDeployment(deployment *extensions.Deployme
 	// TODO: Using client for now, update to use store when it is ready.
 	return dc.expClient.Deployments(deployment.ObjectMeta.Namespace).Update(deployment)
 }
+
+func (dc *DeploymentController) rollbackToTemplate(deployment *extensions.Deployment, spec api.PodSpec) (*extensions.Deployment, error) {
+	if !api.Semantic.DeepEqual(deployment.Spec.Template.Spec, spec) {
+		glog.Infof("Rolling back deployment %s to spec %v", deployment.Name, spec)
+		deployment.Spec.Template.Spec = spec
+	}
```

Comment:
I would want to know that my rollback didn't happen because I tried to roll to the same spec.